eZ/Publish/API/Repository/SearchService.php

```diff
@@ -29,14 +29,34 @@
      * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException if query is not valid
      *
      * @param \eZ\Publish\API\Repository\Values\Content\Query $query
-     * @param array $fieldFilters - a map of filters for the returned fields.
+     * @param array $languageFilter - a map of filters for the returned fields.
      *        Currently supports: <code>array("languages" => array(<language1>,..), "useAlwaysAvailable" => bool)</code>
      *                            useAlwaysAvailable defaults to true to avoid exceptions on missing translations
```

Comment:
Should (for 5.4+) add new line describing that languages is a prioritized list of languages when it comes to search, as for findContent it should also specify it will return fields in those languages by default, but could add new flag there saying that it only return fields for the matched language (this could become default in 6.0 btw as it kind of makes more sense for search).